roles/minio/templates/docker-compose.yml

```diff
@@ -17,6 +17,24 @@ services:
     restart: always
     networks:
       - webproxy
+{% if minio_console_enable %}
+  minio_console1:
+    image: minio/console:{{ minio_console_version }}
+    expose:
+      - 9090
+    environment:
+      CONSOLE_PBKDF_PASSPHRASE: '{{ minio_console_secret_key }}'
+      CONSOLE_PBKDF_SALT: '{{ minio_console_secret_key_salt }}'
+      MINIO_BROWSER_REDIRECT_URL: '{{ minio_console_domain }}'
```

Comment:
add this option to minio container, so people accessing minio_domain are redirected to the minio_console_domain